docs/README.rst

```diff
@@ -0,0 +1,8 @@
+To build a local copy of the dask docs::
+
+  git clone git@github.com:blaze/dask.git
+  cd dask/docs
+  conda create -n daskdocs --file requirements-docs.txt
+  source activate daskdocs
```

Comment:
No, we definitely don't want to force it on anyone. The requirements-docs.txt file was already there, so this seems to be the way people have been doing it so far. Should we write out non-conda instructions or just some note such as "Building the docs does not require conda but can be done quickly with conda with the following instructions:" ?